docs/EVENTS.md

```diff
@@ -6,102 +6,114 @@ and meeting events as follows:
 Remember that you should add the listener after the sdk has been initialized successfully.
 
 ```javascript
-import { ZoomEmitter } from 'react-native-zoom-us';
+import { NativeModules } from 'react-native';
 
-const zoomEmitter = new NativeEventEmitter(ZoomEmitter);
+const zoomEmitter = new NativeEventEmitter(NativeModules.ZoomUs);
```

Comment:
Im not sure the reason, I was having this issue and didnt think that was the reason because as you say, is the same. But after doing that change I stopped getting a weird warning for NativeEventEmitter in the console.

Reply:
I took a look on that and I've fixed the warnings on android in https://github.com/mieszko4/react-native-zoom-us/pull/150